tableschema/VERSION

```diff
@@ -1,2 +1 @@
-1.0.12
-
+1.0.13
```

Comment:
The same as at https://github.com/frictionlessdata/goodtables-py/pull/266#pullrequestreview-111517371